package.json

```diff
@@ -92,6 +93,7 @@
     "jest-environment-jsdom": "^29.6.2",
     "mini-css-extract-plugin": "^2.6.0",
     "mocha": "^10.2.0",
+    "monday-ui-style": "^0.1.196",
```

Comment:
We can't really do this - it will lead to circular dependency - we are using `vibe-storybook-components` to build `monday-ui-style` storybook